README.md

```diff
@@ -33,83 +33,48 @@ line. For additional examples, see the [examples](./examples) directory.
-
-
-   if __name__ == "__main__":
-       parser = argparse.ArgumentParser(
-           description="SignalFx SignalFlow streaming analytics demo"
-       )
-       parser.add_argument(
-           "--stream-endpoint",
-           help="SignalFx SignalFlow stream API endpoint",
-           default="https://stream.signalfx.com",
-       )
-       parser.add_argument("token", help="Your SignalFx API access token")
-       parser.add_argument("program", help="SignalFlow program to execute")
-       options = parser.parse_args()
-
-       client = SignalFlowClient(
-           token=options.token,
-           endpoint=options.stream_endpoint,
-       )
-
-       try:
-           # Execute the computation and iterate over the message stream
-           print("Requesting computation: {0}".format(options.program))
-           c = client.execute(options.program)
-           print("Waiting for data...")
-           for msg in c.stream():
-               if isinstance(msg, messages.DataMessage):
-                   print(
-                       "\033[34;1m{0}\033[;0m @{1}: {2}".format(
-                           "data",
-                           msg.logical_timestamp_ms,
-                           ", ".join(
-                               [
-                                   "\033[;1m{0}\033[;0m: {1}".format(k, v)
-                                   for k, v in msg.data.items()
-                               ]
-                           ),
-                       )
-                   )
-               elif isinstance(msg, messages.EventMessage):
-                   print(
-                       "\033[35;1m{0}\033[34;1m@\033[;0m @{1}: {2}".format(
-                           "event",
-                           msg.timestamp_ms,
-                           ", ".join(
-                               [
-                                   "\033[;1m{0}\033[;0m: {1}".format(k, v)
-                                   for k, v in msg.properties.items()
-                               ]
-                           ),
-                       )
-                   )
-               elif isinstance(msg, messages.MetadataMessage):
-                   print(
-                       "\033[32;1m{0}\033[;0m for \033[;1m{1}\033[;0m:\n  {2}".format(
-                           "metadata",
-                           msg.tsid,
-                           "\n  ".join(
-                               [
-                                   "\033[;1m{0}\033[;0m: {1}".format(k, v)
-                                   for k, v in msg.properties.items()
-                               ]
-                           ),
-                       )
-                   )
-       except KeyboardInterrupt:
-           print(" Detaching from computation...")
-       finally:
-           client.close()
-       print("Done.")
-   ```
+```python
+#!/usr/bin/env python
+
+import argparse
+
+import signalfx
+from signalfx.signalflow import messages, SignalFlowClient
```

Comment:
It looks like signalfx and messages aren't being used in the script.

Reply:
You're right. The unnecessary imports and #noqa were there before the move so I didn't touch them. Should be fixed now.